WorkflowAdminBundle/Resources/public/less/workflow.less

```diff
@@ -0,0 +1,3 @@
+.workflow-cell-parameter {
+    text-align: center;
```

Comment:
it's weird one part of css for workflow are in open-orchestra-less.css and an other part here